firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -673,7 +674,11 @@ def test_update_success_200(self):
         assert response_data["id"] == str(instance.id)
 
         self.assert_updated_instance(data, self.get_updated_instance(instance))
-        self.assert_update_response(data, response_data)
+
+        if self.model in [Audio, Image, Video]:
```

Comment:
Why is this type check necessary? The base class should not need to know about the specific types that are using it

Reply:
You should be able to override assert_update_response in the base media test, I think